STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/inc/_az_ulib_dm_blob_ustream_interface.h

```diff
@@ -0,0 +1,52 @@
+#ifndef AZ_ULIB_DM_BLOB_USTREAM_INTERFACE_H
+#define AZ_ULIB_DM_BLOB_USTREAM_INTERFACE_H
+
+#include "az_ulib_result.h"
+#include "az_ulib_ustream.h"
+#include "azure/az_core.h"
+#include "nx_api.h"
+#include "nx_web_http_client.h"
+#include "nx_wifi.h"
+#include "nxd_dns.h"
+#include "stm_networking.h"
+#include "wifi.h"
```

Comment:
You probably don't need all these includes in the `.h`. Keep in the `.h` only the ones that you need to define the signatures.